webapp/channels/src/components/team_settings/team_settings.tsx

```diff
@@ -36,6 +39,7 @@ const TeamSettings = ({
                 <GeneralTab
                     team={team}
                     activeSection={activeSection}
+                    intl={intl}
```

Comment:
Why is this needed? Shouldn't injectIntl inject it itself without us passing it?